src/SearchForm.cpp

```diff
@@ -251,6 +251,8 @@ void SearchResultList::MessageReceived(BMessage* message) {
 			if (e == NULL) {
 				SearchResultList_Data* itemData = new SearchResultList_Data();
 				AddItem(new BStringItem("No matches found!"), 0);
+				// Disable this item so the user can't click on it.
+				SetEnabled(false);
```

Comment:
If you do it this way, you disable the whole list. This is fine, but be sure to re-enable it when the user makes another search?

Otherwise do it just on the item:

BStringItem* item = new BStringItem(...);
item->SetEnabled(false);
AddItem(item);

Reply:
Yay! Finished (I think).